guidelines.md

```diff
@@ -84,7 +97,7 @@ The authentication below will connect with the demo enterprise that is configure
 * **ClientToken** - `E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D`
 * **AccessToken** - `C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D`
 
-The enterprise is based in the United Kingdom, it accepts `GBP`, `EUR` and `USD` currencies (any of them may be used). As a tax rate, either `0.0`, `0.05` or `0.20` can be used.
+The enterprise is based in the United Kingdom, it accepts `GBP`, `EUR` and `USD` currencies (any of them may be used). 
```

Comment:
Why were the tax rates removed?

Reply:
Working on adding a Taxation section on the same page to address this YT: https://mews.myjetbrains.com/youtrack/issue/RND-3849

so it should mention:
- integration partner should download the enterprise info, tax environments and update the list of taxes they can use.
- only use Tax Rate codes, and not numeric tax rates (the deleted part)
- note the applicable ValidityStart- and ValidityEndUtc as well as any government announcements to anticipate changes to tax rates